### ISSUE TITLE
Invisible column

### ISSUE THREAD

**Issue by OswaldoFaccioni** [thansen-solire/datatables-light-columnfilter#6]
First of all, sorry about my english.

The first column of my table is invisible, when I execute the code I realize one more column in filter row.
I'm using angular datatables.

Create the columns filter

```
.withLightColumnFilter({
'0': {
    type: 'text'
},
'1': {
    type: 'text'
},
'2': {
    type: 'text'
},
'3': {
    type: 'text'
},
'4': {
    type: 'text'
}
})
```

Create the columns to my table
`
DTColumnBuilder.newColumn('Id').withTitle('Id').notVisible(),
DTColumnBuilder.newColumn('Nome').withTitle('Nome'),
DTColumnBuilder.newColumn('Prazo').withTitle('Prazo execução'),
DTColumnBuilder.newColumn('Carteira').withTitle('Carteira')
`
The result is http://s33.postimg.org/y1j3zcprj/error.png.

As you can see, in the table header, there's one more column.

I think that I found the solution, I tested in my project and it worked.

I just added the if condition in file "dataTables.lightColumnFilter.js" after line 134.

```
if (!self.dataTable.column(index).visible()){
            return;
}
```

Could someone check if it's a valid solution for this problem?

**Comment by thansen-solire** [comment]
Sorry for the delay, I'll check today

**Comment by thansen-solire** [comment]
Available in 1.0.0 release
